### PR TITLE
#5980 Detect Browser Firefox Rocket

### PR DESCRIPTION
**Pull request by sanchezzzhak**

### CODE REVIEW

File: Parser/Client/Browser.php [hunk 1]
```diff
@@ -206,7 +207,7 @@ class Browser extends ClientParserAbstract
         'BlackBerry Browser' => array('BB'),
         'Baidu'              => array('BD', 'BS'),
         'Amiga'              => array('AV', 'AW'),
-        'Chrome'             => array('CH', 'BA', 'BR', 'CC', 'CD', 'CM', 'CI', 'CF', 'CN', 'CR', 'CP', 'IR', 'RM', 'AO', 'TS', 'VI', 'PT', 'AS', 'TB', 'AD', 'SB'),
+        'Chrome'             => array('CH', 'BA', 'BR', 'CC', 'CD', 'CM', 'CI', 'CF', 'CN', 'CR', 'CP', 'IR', 'RM', 'AO', 'TS', 'VI', 'PT', 'AS', 'TB', 'AD', 'SB', 'FR'),
```
Comment (sgiehl) [inline Parser/Client/Browser.php]: Is Firefox Rocket really based on Chrome?

Comment (sanchezzzhak) [reply Parser/Client/Browser.php]: I removed the binding to chrome
a quick inspection didn't show me the chromium. https://github.com/mozilla-tw/FirefoxLite

**Comment by Findus23** [comment]
As I explained in https://github.com/matomo-org/device-detector/issues/5980, Firefox Rocket is essentially a webview using the Android system Webkit renderer. So it doesn't use Chromium source code but essentially it is using Chromium to render the page.
I wouldn't add it to the chrome-base list, but to the webkit one.

**Review by sgiehl — APPROVED** [review]
LGTM. 

@Findus23  should we add it to the Firefox browser family? The rendering engine is detected separately.

**Comment by Findus23** [comment]
Hm, it makes sense for people to expect this category as "Browsers Mozilla develops under the Firefox brand" and therefore it should be in the Firefox family.

**Pull request merged**